devsetup/Makefile

```diff
@@ -316,3 +310,14 @@ bmaas_cleanup:
 	make bmaas_crc_baremetal_bridge_cleanup
 	make bmaas_crc_attach_network_cleanup
 	make bmaas_network_cleanup
+
+.PHONY: ansibleee_cleanup
+ansibleee_cleanup: edpm_play_cleanup
```

Comment:
This has the same name as the ansibleee_cleanup from Makefile in the root of the repo, which does something completely different. I think this needs to be renamed to indicate we are talking about openstackansibleee resources, and not the operator.